openedx_tagging/core/tagging/rest_api/v1/permissions.py

```diff
@@ -58,3 +58,23 @@ def has_object_permission(self, request, view, obj):
         """
         obj = obj.taxonomy if isinstance(obj, Tag) else obj
         return rules.has_perm("oel_tagging.list_tag", request.user, obj)
+
+    def has_permission(self, request, view):
+        """
+        Returns True if the request user is allowed the given view on the Taxonomy model.
+
+        We override this method to avoid calling our view's get_queryset(), which performs database queries.
+        """
+        # Workaround to ensure DjangoModelPermissions are not applied
+        # to the root view when using DefaultRouter.
```

Comment:
Could you just expand on this comment slightly? As written, I don't _really_ understand it so I wouldn't know how to test if this is working / still needed or not.

Reply:
While addressing your comment, I found a better way to handle this issue: 29e644e

Is that clear enough?

Reply:
Nice :)